translations/messages.en.yaml

```diff
@@ -21,11 +21,21 @@ app:
         managing_your_speakers: Managing your speakers
         name: Name
         new_conference: New conference
+        new_speakers: New speakers
+        new_talks: New talks
         past_event: Past event
+        show_all: Show all
         show_talks: Show talks
         speaker: Speaker
         speakers: Speakers
         starts_at: Starts at
+        statistics:
+            day: Jour
+            lifetime: Durée de vie
+            month: Mois
+            week: Semaine
+            2weeks: 2 Semaines
+            year: Année
```

Comment:
This is english translation file, please translate :smile:

Reply:
ahah, fixed